spec/models/user_spec.rb

```diff
@@ -90,6 +90,20 @@
     end
   end
 
+  describe "#mentor?" do
+    it "is expected to be true when the user has a mentor profile" do
+      user = create(:user, :mentor)
+
+      expect(user.mentor?).to be true
+    end
+
+    it "is expected to be false when the user does not have a mentor profile" do
+      user = create(:user)
+
+      expect(user.mentor?).to be false
+    end
+  end
```

Comment:
This here is a bit interesting - we are actually testing that our factory is working as expected, not the code itself.

Reply:
pretty much half the model tests are basically pointless and will never break imo, but good to have in case the logic becomes more complicated in the future